src/CMakeLists.txt

```diff
@@ -1639,24 +1606,21 @@ else()
                RESOURCE DESTINATION "${_APPDIR}/Resources" )
    else()
 
-      install( TARGETS ${TARGET} RUNTIME )
+      install( TARGETS ${TARGET}
+               RUNTIME
+               RESOURCE DESTINATION "${_PKGDATA}" )
 
       install( DIRECTORY "${_DEST}/${_LIBDIR}/"
                DESTINATION "${_LIBDIR}"
                USE_SOURCE_PERMISSIONS
```

Comment:
Is there a reason you need to keep the source permissions? In the Fedora build we actually want the libraries to be executable, so permissions of 755 would be appropriate. I haven't looked into why it isn't getting the 755 permissions at build time, but this has been an issue in the past because without the executable bit, the debug info extraction doesn't run and there are no symbols for any of the libraries.

Reply:
I'm pretty sure that without it will copy the files with 0666 permissions. This should not be an issue for Fedora though - without Conan this step is effectively an no-op, as all the libraries are installed in the previous step.